tests/utils/test_datetime.py

```diff
@@ -0,0 +1,89 @@
+import pytest
+
+from datetime import datetime
+
+from trailblazer.utils.datetime import (
+    get_date_days_ago,
+    convert_days_to_minutes,
+    convert_timestamp_to_minutes,
+    get_datetime_from_timestamp,
+)
+
+NUMBER_OF_MINUTES_IN_TWO_DAYS: int = 2880
+MAX_NR_MINUTES_IN_TIMESTAMP: int = 1439
```

Comment:
I am a bit confused by these variables.

Should these be moved to a constants module? Or are they supposed to be used like fixtures and be in conftest?

Reply:
They are only used in the test (so not in the constants.py) and unlikely to be reused as they are very specific for these datetime tests, hence they should not pollute a conftest.py. I placed directly in the test module to avoid magic numbers.